test/Microsoft.AspNet.Razor.Test/Framework/ParserTestBase.cs

```diff
@@ -398,6 +398,20 @@ private static void EvaluateTagHelperBlock(ErrorCollector collector, TagHelperBl
             }
             else
             {
+                if (!string.Equals(actual.TagName, expected.TagName, StringComparison.Ordinal))
+                {
+                    collector.AddError(
+                        "{0} - FAILED :: TagName mismatch for TagHelperBlock :: ACTUAL: {1}",
+                        expected.TagName,
+                        actual.TagName);
+                }
+
+                if (actual.SelfClosing != expected.SelfClosing)
```

Comment:
for both this and the above comparison, put expected on the left (as in `Assert.Equal()`)